templates/hello.html.twig

```diff
@@ -0,0 +1,20 @@
+{% extends "layout.html.twig" %}
+{% block content %}
+    <div class="column1" style="margin-top: 10px;height: 50%;margin-left: 25%;width: 50%">
+        <div class="card" style="background-color: #17a2b8">
+            <form action="multiplication.php" method="post">
```

Comment:
Why `multiplication.php`?

Reply:
What should it be?I also did not get it

Reply:
I am asking you why does this post request hanld by multiplication.php page

Reply:
Do you know what does do `action` attribute in `form` attribute?